src/components/data/TableContext.ts

```diff
@@ -1,13 +1,20 @@
 import { createContext } from 'preact';
 import type { RefObject } from 'preact';
 
-export type TableInfo = {
+export type CommonTableInfo = {
   /** This table has click-able, focus-able rows */
-  interactive: boolean;
+  interactive?: boolean;
   /** This table has a sticky header */
-  stickyHeader: boolean;
+  stickyHeader?: boolean;
   /** Turn off outer table borders */
-  borderless: boolean;
+  borderless?: boolean;
+  /** Do not show a different background every other row */
+  unStriped?: boolean;
```

Comment:
Probably `striped` with a default value of `true` would be easier to reason about, but this follows our rule that boolean props should have a `false` default value, so that we don't have to overwrite them as `striped={false}`.

That said, we could define a `striped` prop with value false, and then add it everywhere we are using tables, even if it's a breaking change.

Reply:
> Probably striped with a default value of true would be easier to reason about, but this follows our rule that boolean props should have a false default value

The original rationale for it is that you can write something like `<button disabled>`, as in HTML. In that case it is quite easy to think about "the disabled state". This only works if the property name has a good one-word opposite though. Once the opposite needs a prefix like "no" or "un" this gets into the realm of double-negatives, which I find confusing to think about. I would find it more readable to just have `stripes={false}` in this case.

Reply:
Fully agree. I have just pushed the changes to use `striped` with default value `true` instead of `unstriped` with default value `false`.